### PR TITLE
Broadcast attri shape->vector<size_t>

### PR DESCRIPTION
**Pull request by scxiao**

### CODE REVIEW

File: src/include/migraphx/op/broadcast.hpp [hunk 1]
```diff
@@ -34,31 +34,29 @@ struct broadcast
         return pack(f(self.axis, "axis"));
     }
 
-    shape broadcast_shape;
+    std::vector<std::size_t> broadcast_lens;
```
Comment (pfultz2) [inline src/include/migraphx/op/broadcast.hpp]: We should probably add this field to the reflect method.

Comment (scxiao) [reply src/include/migraphx/op/broadcast.hpp]: What field should we add? keeping the current implementation?

Comment (scxiao) [reply src/include/migraphx/op/broadcast.hpp]: Fixed

**Comment by pfultz2** [comment]
Actually, this same change should be applied to `op::scalar` as well.

**Pull request merged**